test/testE2E.t.sol

```diff
@@ -817,4 +821,29 @@ contract E2ETests is Test {
         // check disputed reporter balance
         assertEq(token.balanceOf(address(fakeDisputedReporter)), _bobBalance + fakeSlashAmount);
     }
+
+    function testNoVotesForDispute() public {
+        // no votes on a dispute
+
+        // stake for parachain
+        vm.startPrank(bob);
+        token.approve(address(staking), fakeStakeAmount);
+        staking.depositParachainStake(
+            fakeParaId, // _paraId
+            bytes("consumerChainAcct"), // _account
+            fakeStakeAmount // _amount
+        );
+        vm.stopPrank();
+
+        // begin initial dispute
+        uint256 _startVote = block.timestamp;
+        vm.prank(paraOwner);
+        gov.beginParachainDispute(
+            fakeQueryId, fakeTimestamp, fakeValue, fakeDisputedReporter, fakeDisputeInitiator, fakeSlashAmount
+        );
+
+        // todo: what is supposed to be tested here?
```

Comment:
If no votes are cast, it should resolve to `invalid` and everyone should get refunded